src/tests/ports/catalog.spec.ts

```diff
@@ -244,6 +244,10 @@ test('catalog component', function () {
           ],
           total: 1,
         })
+        console.log(
+          'dbClientQueryMock.mock.calls: ',
+          dbClientQueryMock.mock.calls
+        )
```

Comment:
Would you mind removing this `console.log` if not relevant 🙏 ?

Reply:
sure, done!